src/main/java/org/sagebionetworks/web/client/DisplayUtils.java

```diff
@@ -285,8 +285,8 @@ public static void showNotification(String message, String href, String buttonTe
 		if (timeout != null) {
 			settings.setDelay(timeout);
 		} else {
-			// TODO: For Entity Finder testing, we set notifications to stay open for 30s by default.
-			settings.setDelay(30 * 1000);
+			// By default, notifications stay open for 15s
+			settings.setDelay(15 * 1000);
```

Comment:
Halving the value, but making this change permanent after conversation with Duminda